scripts/init.groovy.d/__bootstrap_jenkins_autoconfig.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version='1.1' encoding='UTF-8'?>
+<project>
+  <actions/>
+  <description>Initial bootstrap job that makes the jenkins controller bootstrap itself. Should only be ran on a first intall of the jenkins controller.</description>
+  <displayName>First Boot Jenkins Auto-Bootstrap</displayName>
+  <keepDependencies>false</keepDependencies>
+  <properties>
+    <hudson.plugins.jira.JiraProjectProperty plugin="jira@3.1.3"/>
+    <com.synopsys.arc.jenkinsci.plugins.jobrestrictions.jobs.JobRestrictionProperty plugin="job-restrictions@0.8"/>
+    <hudson.model.ParametersDefinitionProperty>
+      <parameterDefinitions>
+        <hudson.model.StringParameterDefinition>
+          <name>JENKINS_USER</name>
+          <description>Github username of admin</description>
+          <defaultValue></defaultValue>
+          <trim>false</trim>
+        </hudson.model.StringParameterDefinition>
+        <hudson.model.PasswordParameterDefinition>
+          <name>JENKINS_PASSWORD</name>
+          <description>Administrators github Personal Access Token with scopes (user:email, repo, read:org)</description>
+          <defaultValue>{AQAAABAAAABAIhhGLd2ww/D30Fh5V0Tb02PKJiOvpaEnU0Cu3wzxo7Z0JzxLoN8uoRMcW12k16ppPP5KVh86hAJkmv/DkLVHX6fhSkHQMWKMZcYpNYtcx+8=}</defaultValue>
```

Comment:
Is it necessary to have this `defaultValue` here? Unsure how sensitive this is, but just noting that anyone with access to the Jenkins controller or script console could decrypt this. I think that's just our team though, so this probably isn't a blocker.